.github/workflows/linter.yml

```diff
@@ -0,0 +1,61 @@
+name: Lint/Sec Checks
+
+on: 
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    branches: [main]
+
+jobs:
+  go-linters:
+    name: Go linters
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check code formatting
+        uses: golangci/golangci-lint-action@v3
+        with:
+          # https://golangci-lint.run/usage/linters/
+          args: --disable-all --enable goimports
+
+      - name: Check code complexity
+        uses: golangci/golangci-lint-action@v3
+        with:
+          args: --disable-all --enable funlen --enable gocyclo 
+
+      - name: Check code style and patterns
+        uses: golangci/golangci-lint-action@v3
+        with:
+          args: --disable-all --enable dupl --enable revive --enable whitespace
+
+      - name: Check bugs and unused code
+        uses: golangci/golangci-lint-action@v3
+        with:
+          args: --disable-all --enable errcheck --enable unused --enable staticcheck 
+
+      - name: Check code performance
+        uses: golangci/golangci-lint-action@v3
+        with:
+          args: --disable-all --enable maligned --enable prealloc
+
+  gosec:
```

Comment:
Same.. one job is `go-linters`, the other is `gosec`, etc. Let's please make these files internally consistent, and also externally consistent with other workflows.

Reply:
Updated.